core/common/src/main/java/alluxio/util/ShellUtils.java

```diff
@@ -134,6 +136,54 @@ public static UnixMountInfo parseMountInfo(String line) {
     return builder.build();
   }
 
+  /**
+   * Checks whether a path is the mounting point of a RAM disk volume.
+   *
+   * @param path  a string represents the path to be checked
+   * @param fsTypes an array of strings represents expected file system type
+   * @return true if the path is the mounting point of volume with one of the given fsTypes,
+   *         false otherwise
+   * @throws IOException if the function fails to get the mount information of the system
+   */
+  public static boolean isMountingPoint(String path, String[] fsTypes) throws IOException {
+    List<UnixMountInfo> infoList = getUnixMountInfo();
+    for (UnixMountInfo info : infoList) {
+      Optional<String> mountPoint = info.getMountPoint();
+      Optional<String> fsType = info.getFsType();
+      if (mountPoint.isPresent() && mountPoint.get().equals(path) && fsType.isPresent()) {
+        for (String expectedType : fsTypes) {
+          if (fsType.get().equalsIgnoreCase(expectedType)) {
+            return true;
+          }
+        }
+      }
+    }
+    return false;
+  }
+
+  /**
+   * Checks whether an Alluxio service is running.
+   *
+   * @param className class name of the Alluxio service
+   * @return whether the Alluxio service is running
+   */
+  public static boolean isAlluxioRunning(String className) {
+    String[] command = {"bash", "-c",
+        "ps -Aww -o command | grep -i \"[j]ava\" | grep " + className};
+    try {
+      Process p = Runtime.getRuntime().exec(command);
+      try (InputStreamReader input = new InputStreamReader(p.getInputStream())) {
+        if (input.read() >= 0) {
+          return true;
+        }
+      }
+      return false;
+    } catch (IOException e) {
+      System.err.format("Unable to check Alluxio status: %s.%n", e.getMessage());
+      return false;
+    }
+  }
+
```

Comment:
is this code also copied or freshly-written?

Reply:
They are copied from the old `Utils.java` class, which is removed in this change.